ghost/members-importer/test/MembersCSVImporter.test.js

```diff
@@ -221,6 +221,222 @@ describe('MembersCSVImporter', function () {
             }]);
             should.deepEqual(membersRepositoryStub.update.args[0][1].id, 'test_member_id');
         });
+
+        it('should subscribe or unsubscribe members as per the `subscribe_to_emails` column', async function () {
```

Comment:
A bit verbose and cumbersome, but this one has come up a couple times so I want to just squash it dead.